cranelift/filetests/filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -8,29 +8,57 @@ function %one_arg(i64) windows_fastcall {
 block0(v0: i64):
     return
 }
-; check: function %one_arg(i64 [%rcx], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
-; nextln: ss0 = incoming_arg 16, offset -48
+; check:  function %one_arg(i64 [%rcx], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
+; nextln: ss0 = incoming_arg 16, offset -16
+; check:  block0(v0: i64 [%rcx], v1: i64 [%rbp]):
```

Comment:
Any particular reason to check the rest of the function? Seems independent of making sure there's an `ss0` and that this compiles.

Reply:
I wanted to check the prologues and epilogues for correctness in each function.  The intermediate instructions I'm not as concerned about, although I personally don't mind them being checked unless we think there's a likelihood that this will cause a maintenance headache for these tests.

Reply:
Yeah I'm just thinking of maintenance for these. But I've convinced myself it's actually good to extend these checks here: I don't think we explicitly tested fastcall gpr preservation works as expected anywhere, and these aren't really liable to change in a surprising way.